src/components/SelectedDetail.tsx

```diff
@@ -1,18 +1,16 @@
 import React from "react";
 import * as Config from "../utils/Config";
 import * as Types from "../utils/Types";
-import { getDefaultRowColor } from "../utils/Renderers";
 
 const SelectedDetail = (props: Types.SelectedDetailProps): JSX.Element => {
-  const defaultRowColor = getDefaultRowColor(props.orderedIndex);
   return (
     <thead>
       {Config.markerDetailMap.map((markerDetail, index) => {
         const { key, header, getRowPropContent } = markerDetail;
         const rowProp = props.row[key];
         return (
-          <tr key={index} style={{ background: defaultRowColor }}>
-            <th style={{ background: Config.tableHeaderColor }}>{header}</th>
+          <tr key={index}>
```

Comment:
Related to "Refactored table rows coloring to have dark odd and light even rows"